enginetest/insert_queries.go

```diff
@@ -664,6 +665,23 @@ var InsertScripts = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "insert negative values into auto_increment values",
```

Comment:
Need more robust tests here. Before 0,0,0, run this:

```sql
insert into auto () values ()
```

You should get 31 there, same as for the 0 case.